render/render.js

```diff
@@ -643,22 +643,18 @@ module.exports = function($window) {
 				checkState(vnode, original)
 				onremove(vnode)
 				if (vnode.dom) {
-					var count = vnode.domSize || 1
-					if (count > 1) {
-						var dom = vnode.dom
-						while (--count) {
-							removeNodeFromDOM(dom.nextSibling)
+					var parent = vnode.dom.parentNode
+					if (parent != null) {
```

Comment:
Out of curiosity: how can this happen?

Reply:
I just tried to run the tests without this check. It worked.

Looks like this might only happen if the root-node is detached.

Reply:
Yeah, the only way this can happen is if it's manually removed from the parent between `oncreate` and `ondestroy` *and* is the top node. I'll just remove it, because there's no valid reason to do that anyways.

Reply:
Done.